examples/quadrotor.py

```diff
@@ -52,7 +52,8 @@
 
 # Creating the shooting problem and the FDDP solver
 T = 33
-problem = crocoddyl.ShootingProblem(np.concatenate([hector.q0, np.zeros(state.nv)]), [runningModel] * T, terminalModel)
+q0 = pinocchio.neutral(hector.model)
```

Comment:
we need to revert back the changes in the quadrotor example. The problem was at the example-robot-data repo. See this merged PR: https://github.com/Gepetto/example-robot-data/pull/80